cow_instrument/FlatTree.h

```diff
@@ -32,6 +32,7 @@ class FlatTree {
            std::vector<double> &&pathLengths,
            std::vector<size_t> &&pathComponentIndexes,
            std::vector<size_t> &&detectorComponentIndexes,
+           std::vector<size_t> &&branchNodeComponentIndexes,
```

Comment:
The union of these three index vectors should correspond to the full range, so maybe specifying the latter explicitly would not be necessary.